tests/page_test.go

```diff
@@ -1177,3 +1177,61 @@ func TestPageThrottleNetwork(t *testing.T) {
 		})
 	}
 }
+
+// This test will first navigate to the ping.html site a few times, record the
+// average time it takes to run the test. Next it will repeat the steps but
+// first apply CPU throttling. The average duration with CPU throttling
+// enabled should be longer than without it.
+func TestPageThrottleCPU(t *testing.T) {
+	t.Parallel()
+
+	tb := newTestBrowser(t, withFileServer())
+
+	tb.withHandler("/ping", func(w http.ResponseWriter, req *http.Request) {
+		defer func() {
```

Comment:
Is there any reason this is in a defer?

Reply:
It was due to the linter, which wanted a check on `err` from `req.Body.Close()`. Happy to consider other options.